yarn.lock

```diff
@@ -6531,10 +6546,15 @@ hex-color-regex@^1.1.0:
   resolved "https://registry.yarnpkg.com/hex-color-regex/-/hex-color-regex-1.1.0.tgz"
   integrity sha512-l9sfDFsuqtOqKDsQdqrMRk0U85RZc0RtOR9yPI7mRVOa4FsR/BVnZ0shmQRM96Ji99kYZP/7hn1cedc1+ApsTQ==
 
-highlight.js@^10.5.0:
-  version "10.5.0"
-  resolved "https://registry.yarnpkg.com/highlight.js/-/highlight.js-10.5.0.tgz"
-  integrity sha512-xTmvd9HiIHR6L53TMC7TKolEj65zG1XU+Onr8oi86mYa+nLcIbxTTWkpW7CsEwv/vK7u1zb8alZIMLDqqN6KTw==
+highlight.js@10.7.2:
```

Comment:
@iHiD This is still in the lockfile, but we're not using this. We're using highlight JS 11 as the syntax highlighter for our diffs. I can't seem to remove this from the lockfile even though I specify `--ignore-optional` when adding the `diff2html` library.

Reply:
@joshgoebel Help us oh wise one :)

Reply:
@SleeplessByte Any ideas too?

Reply:
If the version specifiers of both libraries can't be resolved to a single version then you're going to be stuck with two (that's my understanding) since each library will use it's own version.  The best thing to do is to get the upstream to upgrade to version 11.  Perhaps @SleeplessByte has some better ideas though?

Reply:
Thanks, Josh. It seems weird that it's marked as an "optional dependency" but that the `--ignore-optional` flag doesn't do anything.

Reply:
> It seems weird that it's marked as an "optional dependency" 

Well it seems to allow injecting your own highlighter instance (which would make bundling it optional).  One might bring their own custom packaged version, or even a similar library that matched our API, etc... or even just a pre-configured object with many settings/plugins, etc.

Reply:
Yeah. I mean it's an optional dependency in the `yarn.lock` though. Look here: https://github.com/exercism/website/pull/1074/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR5158-R5159

Reply:
Yeah, that's a question for yarn people. :-) I don't know that one.

Reply:
It should have been a _peer_ dependency.

Anyway, this can be solves using yarns _resolutions_. See: https://classic.yarnpkg.com/en/docs/selective-version-resolutions/

You'll want to target `diff2html/highlight.js` For example: `"diff2html/highlight.js": "./node_modules/highlight.js"`